cpp/src/IceBridge/IceBridge.cpp

```diff
@@ -195,10 +195,10 @@ BridgeConnection::outgoingSuccess(ConnectionPtr outgoing)
     //
     // Flush any queued dispatches
     //
-    for (const auto& p : _queue)
+    for (auto& dispatch : _queue)
```

Comment:
I assume const was removed because otherwise `std::move` would not move the members?

Reply:
Yes.